### PR TITLE
playnite: Added Playnite version 4.74

### PR DESCRIPTION
**Pull request by Aetylus**
Created an App Manifest for [Playnite](https://github.com/JosefNemec/Playnite).

### CODE REVIEW

File: bucket/playnite.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      [
+         "PlayniteUI.exe",
+         "Playnite"
+      ]
+   ],
+   "checkver": "github",
+   "autoupdate": {
+      "url": "https://github.com/JosefNemec/Playnite/releases/download/$version/Playnite$version.zip",
+      "hash": {
+         "url": "$url.sha256"
```
Comment (Ash258) [inline bucket/playnite.json]: There is no hash file. Remove whole hash property.

File: bucket/playnite.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   "bin": "PlayniteUI.exe",
+   "shortcuts": [
+      [
+         "PlayniteUI.exe",
+         "Playnite"
+      ]
+   ],
+   "checkver": "github",
+   "autoupdate": {
+      "url": "https://github.com/JosefNemec/Playnite/releases/download/$version/Playnite$version.zip",
```
Comment (Ash258) [inline bucket/playnite.json]:
```suggestion
      "url": "https://github.com/JosefNemec/Playnite/releases/download/$version/Playnite$cleanVersion.zip",
```

File: bucket/playnite.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+   "url": "https://github.com/JosefNemec/Playnite/releases/download/4.74/Playnite474.zip",
+   "hash": "b2cb3c4a34986e9c04d2478f1de29ec94647c9d3673e02203f29be0f5be93ba3",
+   "bin": "PlayniteUI.exe",
+   "shortcuts": [
+      [
+         "PlayniteUI.exe",
+         "Playnite"
+      ]
+   ],
+   "checkver": "github",
```
Comment (Ash258) [inline bucket/playnite.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/JosefNemec/Playnite/"
    }
```

**Comment by Aetylus** [comment]
You're absolutely correct, I should have. Thanks!

**Comment by r15ch13** [comment]
A launcher for launchers? Awesome :grin:

Some directories require persistence though.
```json
"persist": [
    "browsercache",
    "ExtensionsData",
    "library"
]
```

**Comment by Gargauth** [comment]
There are more directories and files that require persistence:

```
"persist": [
    "browsercache",
    "cache",
    "ExtensionsData",
    "library",
    "config.json"
]
```

**Comment by r15ch13** [comment]
Fixed with https://github.com/lukesampson/scoop-extras/commit/fbfe69ba911aa56dbc9dfb293d26d5e3aa4210fd

**Pull request merged**